apps/public/src/app/(general)/bedankt/Bedankt.tsx

```diff
@@ -0,0 +1,52 @@
+'use client'
+
+import { Button, Heading, Link, Paragraph } from '@amsterdam/design-system-react'
+import { Grid } from '@meldingen/ui'
+import NextLink from 'next/link'
+
+type Props = {
+  meldingId: string
+}
+
+export const Bedankt = ({ meldingId }: Props) => (
+  <Grid paddingBottom="large" paddingTop="medium">
+    <Grid.Cell span={{ narrow: 4, medium: 6, wide: 7 }} start={{ narrow: 1, medium: 2, wide: 2 }}>
+      <Heading className="ams-mb--xs">Bedankt</Heading>
+      <Paragraph>Bedankt voor uw melding.</Paragraph>
+      <Paragraph>{`Wij hebben uw melding ontvangen op 21-11-2023 om 17:11. Uw meldnummer is ${meldingId}.`}</Paragraph>
+    </Grid.Cell>
+
+    <Grid.Cell span={{ narrow: 4, medium: 6, wide: 7 }} start={{ narrow: 1, medium: 2, wide: 2 }}>
+      <Heading level={2} size="level-3" className="ams-mb--xs">
+        Wat doen we met uw melding?
+      </Heading>
+      <Paragraph>Meldingen pakken we binnen 3 werkdagen op.</Paragraph>
+    </Grid.Cell>
+
+    <Grid.Cell span={{ narrow: 4, medium: 6, wide: 7 }} start={{ narrow: 1, medium: 2, wide: 2 }}>
+      <Heading level={3} size="level-3" className="ams-mb--xs">
+        Vragen of meer informatie?
+      </Heading>
+      <Paragraph>
+        Neem dan contact met ons op via{' '}
+        <Link href="tel:+3114020" variant="inline">
```

Comment:
I don't think the '+31' part is necessary here.

```suggestion
        <Link href="tel:14020" variant="inline">
```